v1/compile/bundle_compile_bench_test.go

```diff
@@ -0,0 +1,126 @@
+package compile
```

Comment:
The `v1` root package will look a bit weird when OPA 2.0 hits in a billion years. Then we'd have imports such as `github.com/open-policy-agent/opa/v2/v1` and `github.com/open-policy-agent/opa/v2/v2` to contend with 😬.

Underscores in package names are frowned upon by our linter policies, so `v1_sdk` isn't a viable option. Are there other names that would work (`v1sdk`), or is there another root pkg we can put this under (`api/v1`, `sdks/v1` (`sdk` is already taken 😢))?